ambry-messageformat/src/main/java/com.github.ambry.messageformat/MessageFormatRecord.java

```diff
@@ -59,7 +59,7 @@
   public static final short Metadata_Content_Version_V3 = 3;
   public static final int Message_Header_Invalid_Relative_Offset = -1;
 
-  static short headerVersionToUse = Message_Header_Version_V3;
```

Comment:
Add some notes here to remind people how to increase this version? just like you description.